zabbix-probes/zabbix-probes-common/src/main/java/com/indigo/zabbix/utils/CmdbClient.java

```diff
@@ -0,0 +1,147 @@
+//package com.indigo.zabbix.utils;

```

Comment:
As I told you this morning: 

2. I left Cmdb where it was (by foregetting to remove it totally commented from zabbix-probes-common, sorry about that). 

The reason why I didn't want to move it to commons and extend it in openstack-probes was the fact that in the latter there are too many differences and I didn't have time to understand what there was wrong yesterday. Anyway, it can be movede and "un-commented" in client. as I'm doing right now